src/sider/dialogmoter/components/referat/DeltakerBehandlerHeading.tsx

```diff
@@ -0,0 +1,22 @@
+import { MedisinskrinImage } from "../../../../../img/ImageComponents";
+import { Heading, HStack } from "@navikt/ds-react";
+import React from "react";
+
+interface Props {
+  children: string;
+}
+
+export const DeltakerBehandlerHeading = ({ children }: Props) => {
```

Comment:
For denne og de andre `…Heading` komponentene kunne man vurdert å laget en komponent som man gjenbruker. Ikke kjempeviktig.